### PR TITLE
[DOCS] Port connector docs

### PR DESCRIPTION
**Pull request by leemthompo**
# [📸URL preview](https://elasticsearch_bk_112953.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/es-connectors.html)

## Summary

Ports connector docs from Enterprise Search guide to Elasticsearch guide.

### CODE REVIEW

File: docs/reference/connector/docs/connectors-content-extraction.asciidoc [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
++
+[source,bash]
+----
+$ docker run \
+  -p 8090:8090 \
+  -it \
+  -v extraction-service-volume:/app/files \
+  --network "elastic" \
+  --name extraction-service \
+  docker.elastic.co/enterprise-search/data-extraction-service:$EXTRACTION_SERVICE_VERSION
```
Comment (leemthompo) [inline docs/reference/connector/docs/connectors-content-extraction.asciidoc:259]:
```suggestion
  docker.elastic.co/integrations/data-extraction-service:$EXTRACTION_SERVICE_VERSION
```

**Review by leemthompo — COMMENTED** [review]
cc @wangch079

**Comment by danajuratoni** [comment]
LGTM
One question: I liked the initial proposal use a top level like "Ingest content" where we could bring connectors and crawler. Is that something you still see as a possibility in the near future (8.17-8.18)?

**Comment by leemthompo** [comment]
> One question: I liked the initial proposal use a top level like "Ingest content" where we could bring connectors and crawler. Is that something you still see as a possibility in the near future (8.17-8.18)?

We might need the next docs system to be able to nest these connector references that deeply. If we go one more level, we lose all the "On this page" subheadings in the connectors docs in the current asciidoc system.

**Comment by danajuratoni** [comment]
Got it!
That's a bummer

**Comment by danajuratoni** [comment]
@leemthompo I just noticed SP Server has no incremental sync listed on the table, but they are supported.
image

Could we sneak this small fix here?

**Review by seanstory — APPROVED** [review]
rubberstamp

**Comment by leemthompo** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.x|[](https://github.com/elastic/elasticsearch/pull/113771)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**